web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,268 @@
+  name,
+  basketDenom,
+  description,
+  totalAmount,
+  curator,
+  allowedCreditClasses,
+  minStartDate,
+  startDateWindow,
+}) => {
+  const styles = useStyles();
+
+  return (
+    <Box
+      sx={theme => ({
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: `1px solid ${theme.palette.grey[100]}`,
+        backgroundColor: theme.palette.grey[50],
+        backgroundImage: `url(${topoImg})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+      })}
+    >
+      <Section className={styles.content}>
+        <Grid container>
+          <Grid item xs={12} sm={5} className={styles.imageContainer}>
+            <OptimizedImage
+              className={styles.image}
+              src={forestImg}
+              alt={name}
+            />
+          </Grid>
+          <Grid item xs={12} sm={7} className={styles.textContainer}>
+            <Title variant="h1" className={styles.title}>
+              {name}
+            </Title>
+            <Description className={styles.basketDenom}>
+              {basketDenom}
+            </Description>
+            <Description className={styles.basketDescription}>
+              {description}
+            </Description>
+            <OnBoardingCard className={styles.card}>
+              <Grid
+                container
+                rowSpacing={1}
+                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+              >
+                <Grid item xs={12} sm={6}>
+                  <Item label="total amount" data={formatNumber(totalAmount)} />
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                  <Item
+                    label="curator"
+                    data={curator}
+                    // TODO: harcoded url for curator
+                    link={'https://www.regen.network/'}
+                  />
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                  <Item
+                    label="allowed credit classes"
+                    data={allowedCreditClasses}
+                    // TODO: url for credit class
+                    link={'/credit-classes/'}
+                  />
+                </Grid>
+                {minStartDate && (
+                  <Grid item xs={12} sm={6}>
+                    <Item
+                      label="min start date"
+                      data={formatDate(minStartDate)}
+                    />
+                  </Grid>
+                )}
+                {startDateWindow && (
+                  <Grid item xs={12} sm={6}>
+                    <Item
+                      label="start date window"
+                      data={formatDate(startDateWindow)}
+                    />
+                  </Grid>
+                )}
+              </Grid>
+            </OnBoardingCard>
+          </Grid>
+        </Grid>
+      </Section>
+    </Box>
+  );
+};
+
+export { BasketOverview };
```

Comment:
let's either export the function directly ie
```tsx
export const BasketOverview: React.FC<BasketOverviewProps> = ({
```

or move all named exports to the bottom of the file, for consistency

Reply:
ok, i just copied the shape from a page component which seemed neat to me. I personally think I would export as default in this case. I take note to define a convention

Reply:
I hadn't paid enough attention to your comment before, now I realize what you meant. Thanks!